c-dependencies/js-compute-runtime/host_call.cpp

```diff
@@ -1,6 +1,45 @@
 
 #include "host_call.h"
 
+bool handle_fastly_result(JSContext *cx, int result, int line, const char *func) {
+  FastlyStatus status = convert_to_fastly_status(result);
+  return handle_fastly_result(cx, status, line, func);
+}
+
+FastlyStatus convert_to_fastly_status(int result) {
+  switch (result) {
+  case 0:
+    return FastlyStatus::Ok;
+  case 1:
+    return FastlyStatus::Error;
+  case 2:
+    return FastlyStatus::Inval;
+  case 3:
+    return FastlyStatus::BadF;
+  case 4:
+    return FastlyStatus::BufLen;
+  case 5:
+    return FastlyStatus::Unsupported;
+  case 6:
+    return FastlyStatus::BadAlign;
+  case 7:
+    return FastlyStatus::HttpInvalid;
+  case 8:
+    return FastlyStatus::HttpUser;
+  case 9:
+    return FastlyStatus::HttpIncomplete;
+  case 10:
+    return FastlyStatus::None;
+  case 11:
+    return FastlyStatus::HttpHeadTooLarge;
+  case 12:
+    return FastlyStatus::HttpInvalidStatus;
+  default:
+    MOZ_ASSERT_UNREACHABLE("coding error");
+    return FastlyStatus::None;
```

Comment:
Should we make a `Unknown` enum variant for this case instead of re-using `None`?